### DISCUSSION TITLE
Build and test for Linux/ARM64

### DISCUSSION THREAD

**Discussion by odidev** [curl/curl#15390] [General]
## Description  

For aarch64 platform, I have built "Curl" from source.  

## Steps   

Following [steps](https://github.com/google/re2/blob/main/README) were used for the building and installation:  

``` 
# sudo apt update 
# sudo apt install build-essential autoconf libtool libssl-dev libz-dev make cmake git wget -y 
# wget https://github.com/curl/curl/releases/download/curl-7_43_0/curl-7.43.0.tar.gz 
# tar -xvf curl-7.43.0.tar.gz 
# cd curl-7.43.0 
# ./buildconf 
# ./configure --with-openssl 
# make -j$(nproc) 
# sudo make install 
```  

## Outcome  

To validate that the build was successful we ran a version check. The version info returned matched the expected value.  

All this information is also added to [Software Ecosystem Dashboard for Arm](https://www.arm.com/developer-hub/ecosystem-dashboard/).

**Comment by bagder** [comment]
1. curl works and is tested on aarch64 all the time.
2. 7.43.0 is a mighty old version that we do not recommend anyone uses
